packages/trigger/src/lib/data-mapper.ts

```
@@ -0,0 +1,630 @@
+import type { SupabaseClient } from "@supabase/supabase-js";
+import type { Database } from "./database.types";
+import type { CoachRow } from "./excel-parser";
+import {
+	derivePrimarySpecialty,
+	mapDivisionCode,
+	nullifyEmptyString,
+	parseEventGroups,
+	parseFloat,
```

Biome (2.1.2)

[error] Do not shadow the global "parseFloat" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)